cmd/installer/pack.go

```diff
@@ -27,8 +27,8 @@ import (
 type PackType struct {
 	xml.PdscTag
 
-	// isPublic tells whether the pack exists in the public index or not
-	isPublic bool
+	// IsPublic tells whether the pack exists in the public index or not
+	IsPublic bool
```

Comment:
Why did you decide to capitalize IsPublic but isInstalled begins with a lower case. What is the coding guidelines you follow?

Reply:
Originally that was not intended to be exported, but it turned out to be necessary to check in tests. I was trying to export only the necessary fields, but eventually I think I'll converge to all variables there being exported (capitalized first letter).

Reply:
Ah, sorry for my lack of knowledge of the GO language. Thanks for explaining.